maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/service/PassportAssessmentService.java

```diff
@@ -31,6 +32,13 @@ public PassportAssessmentDTO update(UpdatePassportAssessment updatePassportAsses
         PassportAssessmentDTO passportAssessmentDTO =
                 passportAssessmentMapper.updatePassportAssessmentToPassportAssessmentDTO(updatePassportAssessment);
         log.info("Updating existing passport assessment record");
+        PassportAssessmentEntity existingPassportAssessmentEntity = passportAssessmentImpl.find(passportAssessmentDTO.getId());
+        if(existingPassportAssessmentEntity == null) {
+            throw new ValidationException(String.format("Passport assessment with id %s not found !",passportAssessmentDTO.getId()));
+        }
+        if(existingPassportAssessmentEntity.getPastStatus().equals(STATUS_COMPLETE)) {
+            throw new ValidationException("User cannot modify a completed assessment");
+        }
```

Comment:
There is a missing check here that verifies the assessment has not been modified by another user.

Reply:
This seems to be part of PL/SQL so will have to be addressed as part of migrating that code. This check is not present in FinancialAssessments and IOJ Appeals services either